src/transaction/log_impl.h

```diff
@@ -954,7 +954,8 @@ struct log_header
   LOG_LSA bkup_level2_lsa;	/* Lsa of backup level 2 */
   char prefix_name[MAXLOGNAME];	/* Log prefix name */
   bool has_logging_been_skipped;	/* Has logging been skipped ? */
-  VACUUM_LOG_BLOCKID vacuum_last_blockid;	/* Last processed blockid needed for vacuum. */
+  char vacuum_last_blockid_buf[sizeof (VACUUM_LOG_BLOCKID)];	/* Last processed blockid needed for vacuum. */
```

Comment:
I think we need a dummy byte to align vacuum_last_blockid_buf  as 8 bytes boundary:
```c
  bool has_logging_been_skipped;	/* Has logging been skipped ? */
  bool dummy;
  char vacuum_last_blockid_buf[sizeof (VACUUM_LOG_BLOCKID)];	/* Last processed blockid needed for vacuum. */
```